src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -1,18 +1,34 @@
-import React from 'react';
+// @flow
+import * as React from 'react';
 import moment from 'moment';
 
 import get from '../../../platform/utilities/data/get';
 
 import { RESERVE_GUARD_TYPES } from './constants';
+
+
+// Would be good to pull these type definitions into a common file
```

Comment:
Not necessarily in `src/custom-types/`, but an app-wide file or something, I'm thinking.